web/public/_headers

```diff
@@ -0,0 +1 @@
+/api/*  http://34.227.42.195/api/:splat  200
```

Comment:
I think you want `Access-Control-Allow-Origin: "*"`.

Reply:
Oops looks like I copied the _redirects file into it, fixed now.